receiver/receivercreator/README.md

```diff
@@ -3,7 +3,7 @@
 | Status                   |                       |
 |--------------------------|-----------------------|
 | Stability                | [beta]                |
-| Supported pipeline types | metrics               |
```

Comment:
Please update the component to automatically generate the header

Reply:
Can we do this in a separate PR please? trust me, we'll do it.

Reply:
Went ahead and did so after rebasing